### DISCUSSION TITLE
preguntica pequeña?

### DISCUSSION THREAD

**Discussion by mafer23** [Launch-X-Latam/MisionFrontEnd#452] [Team Posts]
una pregunta debe ser obligatorio hacerlo con vue en frontend , no se puede utilizar angular

**Comment by Darry96** [comment]
Una vez menciono algo que podría cambiarlo a angular u otra cosas que quisiéramos y que haría un encuestas pero abra que esperar haber si al momento lo cambia yo en lo personal también quisiera angular

**Comment by rpajares** [comment]
Yo quiero aprender los dos, aunque veo que este curso, el mayor porcentaje
se va a realizar solo con html.

**Comment by ManuelEGC** [comment]
Buenas noches, les sugiero que dejen de responder los correos por favor. Ya que se envian a todos los correos y llenan de spam.
Gracias compañeros Explorers.

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: rpajares ***@***.***>
Enviado: Friday, October 14, 2022 11:14:12 PM
Cc: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] preguntica pequeña? (#21)


Yo quiero aprender los dos, aunque veo que este curso, el mayor porcentaje
se va a realizar solo con html.

**Comment by Sebastian-Longhitano** [comment]
Me gustaría utilizar REACT o ANGULAR!

**Comment by dibot55** [comment]
Creo que el Sherpa quiere que usemos Vue por que permite crear aplicaciones escalables (osea desde una app pequeña mejorarla poco a poco). En cambio usar React y Angular es crear LA aplicación, es decir una bien hecha como una red social o una app tipo uber.
